modules/navigation/godot_navigation_server.cpp

```diff
@@ -248,7 +248,7 @@ TypedArray<RID> GodotNavigationServer::map_get_regions(RID p_map) const {
 	const NavMap *map = map_owner.get_or_null(p_map);
 	ERR_FAIL_COND_V(map == nullptr, regions_rids);
 
-	const LocalVector<NavRegion *> regions = map->get_regions();
+	const LocalVector<NavRegion *> &regions = map->get_regions();
```

Comment:
Copy introduced in #63584 when attempting to reduce allocations by pre-sizing the target.

Reply:
My mistake, I shouldn't have approved that PR, I did notice and flag the incorrect use in https://github.com/godotengine/godot/pull/63584#discussion_r932544452 but I suspect author was new to c++ at the time, and I guess I assumed these had been fixed.

Reply:
The other copies from the free command were already removed in #70773. I don't see any other instances of copying localvector (or other non-primitive types) in that PR.

For navigation code in general I only found https://github.com/godotengine/godot/commit/4b931fc2696adf25797502653bef4d900d54c3c2#diff-1c9ca6e61a93b480285c3ae7d3b0a4111f8a2e1f045c1299293953a4ee0f5ba8R663 to copy a localvector, but I don't think that use can easily be replaced.

There is no 3.x version of this for now (as I generally don't use it) but I can make a version for that as well.